src/overlays/actors/ovl_En_Kusa/z_en_kusa.h

```diff
@@ -8,12 +8,18 @@ struct EnKusa;
 
 typedef void (*EnKusaActionFunc)(struct EnKusa*, GlobalContext*);
 
+typedef enum {
+    /* 0 */ ENKUSA_TYPE_0,
+    /* 1 */ ENKUSA_TYPE_1,
+    /* 2 */ ENKUSA_TYPE_2
+} EnKusaType;
+
 typedef struct EnKusa {
     /* 0x0000 */ Actor actor;
     /* 0x014C */ EnKusaActionFunc actionFunc;
     /* 0x0150 */ ColliderCylinder collider;
     /* 0x019C */ s16 timer;
-    /* 0x019E */ s8 kusaTexObjIndex;
+    /* 0x019E */ s8 objBankIndex;
```

Comment:
think itd be a good idea to avoid having members named the same as one in the base actor struct